shopBuddy/shopBuddy/AddItemViewController.m

```diff
@@ -0,0 +1,62 @@
+//
+//  AddItemViewController.m
+//  shopBuddy
+//
+//  Created by Rachna Gupta on 7/12/22.
+//
+
+#import "AddItemViewController.h"
+#import "APIManager.h"
+#import "Item.h"
+#import "ItemDetailViewController.h"
+
+@interface AddItemViewController ()
+
+@end
+
+@implementation AddItemViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    [[APIManager shared] getItem:^(NSDictionary *itemDetails, NSError *error) {
+        NSLog(@"😎😎😎 Successfully loaded itemDetails");
+        if (itemDetails) {
+            NSLog(@"😎😎😎 Successfully loaded itemDetails");
+            self.item = [[Item alloc] initWithDictionary:itemDetails[@"products"][0]];
+            
+        }
```

Comment:
1. Can you show an example of the API response?
2. I think is more clear if you initialize the model by extracting individual fields from the response dictionary one by one and passing to the init method of the `Item` model.
3. As suggested above you should have your parsing logic in the networking layer.